### PR TITLE
Fix bug #1069 and fix download onceConnected call

### PR DESCRIPTION
**Pull request by MikeWillis**
Just replacing \ with / on localPath for now.

There seems to be no way to ask the FTP server what the preferred directory separator is, see https://stackoverflow.com/questions/37411642/how-to-get-ftp-servers-file-separator. As noted there, most FTP servers will use / however some do not. Perhaps in the future we should add an option to .ftpconfig to specify the FTP server's preferred directory separator, with / being the default. That way in obscure situations the user can specify whatever separator they want.

I also noticed that it wasn't triggering the download after connecting (if not connected at time of request), so put in a quick fix for that as well

### CODE REVIEW

File: lib/menus/commands.js [hunk 1]
```diff
@@ -391,7 +391,7 @@ const init = () => {
         $treeSelected.each((key, elem) => {
           const path = elem.getPath ? elem.getPath() : '';
           const localPath = path.replace(client.root.local, '');
-          const remotePath = Path.posix.normalize(atom.project.remoteftp.root.remote + localPath);
+          const remotePath = Path.posix.normalize(atom.project.remoteftp.root.remote + localPath.replace(/\\/g, '/'));
```
Comment (icetee) [inline lib/menus/commands.js]: I'd rather suggest this
```javascript
const remotePath = Path.posix.normalize((atom.project.remoteftp.root.remote + localPath).replace(/\\/g, '/'));
```

**Review by icetee — COMMENTED** [review]
You think?

File: lib/menus/commands.js (hunk 1 above)
Comment (MikeWillis) [reply lib/menus/commands.js]: I agree, that probably can't hurt. On my machine atom.project.remoteftp.root.remote was using / already, but it's not a bad idea to replace any \ with / there as well

Comment (icetee) [reply lib/menus/commands.js]: Okay, replace and I accept PR.

Comment (MikeWillis) [reply lib/menus/commands.js]: all set, file is updated

**Comment by icetee** [comment]
I'm sorry to hang you. Next time, modify the last commit. There are not so many changes that should be justified.

**Comment by MikeWillis** [comment]
No worries, and thanks! Actually I'm just learning github so commits etc are all very new to me, I didn't know there was a way to modify a commit

**Pull request merged**